docs/production.md

```diff
@@ -25,3 +24,5 @@ The system is Dockerized into a number of components:
 * `docker compose down` to bring the system down
```

Comment:
Could you add the --tail=## tag to the documentation as well? It may be helpful for the next class to know.